springfield/firefox/templates/firefox/includes/download-unsupported.html

```diff
@@ -23,13 +23,33 @@
   <p>{{ ftl('download-button-please-download-esr') }}</p>
   <div class="download-platform-list">
     <p>
-      <a class="mzp-c-button mzp-t-product download-link os_win64 ga-product-download" href="https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=win64&lang={{ win_download_lang }}" data-download-version="win64" data-cta-text="Download Firefox Extended Support Release 64-bit" data-cta-type="firefox">
+      <a class="{% if flare_styles %}fl-button button-secondary{% else %}mzp-c-button mzp-t-product{% endif %} download-link os_win64 ga-product-download"
+         href="https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=win64&lang={{ win_download_lang }}"
+         data-download-version="win64"
+         data-cta-text="Download Firefox Extended Support Release 64-bit"
+         data-cta-type="firefox">
         {{ ftl('download-firefox-esr-64') }}
+        {% if flare_styles %}
+          <include:icon
+            icon_name="arrow-down"
+            hidden=true
+            extra_class="fl-icon-right" />
+          {% endif %}
       </a>
     </p>
     <p>
-      <a class="mzp-c-button mzp-t-product download-link os_win ga-product-download" href="https://download.mozilla.org/?product=firefox-esr115-latest-ssl&os=win&lang={{ win_download_lang }}" data-download-version="win" data-cta-text="Download Firefox Extended Support Release 32-bit" data-cta-type="firefox">
+      <a class="{% if flare_styles %}fl-button button-secondary{% else %}mzp-c-button mzp-t-product{% endif %} download-link os_win ga-product-download"
```

Comment:
Another instance of the mzp classes that might not belong. I think there are a few more. Rather than commenting on each one, it might be good to search/replace.